extension/src/popup/ducks/transactionSubmission.ts

```diff
@@ -245,6 +301,18 @@ const transactionSubmissionSlice = createSlice({
     saveAllowedSlippage: (state, action) => {
       state.transactionData.allowedSlippage = action.payload;
     },
+    startHwConnect: (state) => {
+      state.hardwareWalletData.status = HwOverlayStatus.IN_PROGRESS;
+      state.hardwareWalletData.transactionXDR = "";
```

Comment:
clearing the transactionXDR just to be safe, since that's how the component will know the intent is to import an account and not sign